homeassistant/components/owlet/strings.json

```
@@ -0,0 +1,58 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "region": "Region",
+          "username": "Email",
+          "password": "Password"
```

```suggestion
          "password": "[%key:common::config_flow::data::password%]"
```

use references where possible. Please adopt other places too